docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
```

Comment:
on local machine I also have
```
    networks:
      - gratheon
networks:
  gratheon:
```
for example here in
https://github.com/Gratheon/image-splitter/blob/main/docker-compose.dev.yml#L23

This allows docker containers to work within a same network so that you could make requests to schema-registry by its name
![Screenshot 2024-05-25 at 23 59 18](https://github.com/Gratheon/telemetry-api/assets/445122/be55f0bf-86ec-4dcd-a6c3-e78cd836c9a0)